modules/cnv-subscribing-to-the-catalog.adoc

```diff
@@ -20,20 +20,13 @@ Operators.
 
 . Read the information about the Operator and click *Install*.
 
-. On the *Create Operator Subscription* page:
-.. For *Installed Namespace*, ensure that the *Operator recommended namespace* option
-is selected. This installs the Operator in the mandatory `openshift-cnv` namespace, which
-is automatically created if it does not exist.
-+
-[WARNING]
-====
-Attempting to install the {CNVProductName} Operator in a namespace other than
-`openshift-cnv` causes the installation to fail.
-====
 .. Select *{CNVVersion}* from the list of available *Update Channel* options.
 .. For *Approval Strategy*, ensure that *Automatic*, which is the default value,
 is selected.
 {CNVProductNameStart} automatically updates when a new z-stream release is
 available.
 
-. Click *Subscribe* to make the Operator available to the `openshift-cnv` namespace.
+. Click *Subscribe*.
++
+On the *Installed Operators* screen, the *Status* displays *Succeeded* when
+{CNVProductNameStart} has finished installing.
```

Comment:
s/{CNVProductNameStart}/{CNVProductName} 

(`container-native virtualization` isn't capitalized unless it's at the beginning of a sentence or when specifically referring to an instance where it's capitalized in the UI)